SECURITY.md

```diff
@@ -2,19 +2,17 @@
 
 ## Supported Versions
 
-The strategy of support of the Otter Framework is the following one:
+A given major version of Otter is officially supported as long as it relies on an Angular version that is supported by Google (see [supported Angular versions](https://angular.io/guide/releases#actively-supported-versions)).
 
-- Only the **3 last Major versions** are supported.
```

Comment:
@nhoffmann-1A Do we have our releasing strategy to be shared in this file?

Reply:
@nhoffmann-1A  Maybe you can share here the guidelines that would need to be set in this PR?